src/Core/Components/DataGrid/FluentDataGrid.razor.cs

```diff
@@ -541,10 +576,7 @@ public void SetLoadingState(bool loading)
     /// (either <see cref="Items"/> or <see cref="ItemsProvider"/>).
     /// </summary>
     /// <returns>A <see cref="Task"/> that represents the completion of the operation.</returns>
-    public async Task RefreshDataAsync()
-    {
-        await RefreshDataCoreAsync();
-    }
+    public Task RefreshDataAsync() => RefreshDataCoreAsync();
```

Comment:
Using expression body is fine, but why remove the async / await? Think we should leave that in.

Reply:
Async/await is unnecessary here and removing it doesn't change behavior but prevents the generation of an unnecessary underlying state machine by the compiler. Any way, I will remove it from this PR.